extensions/common/store/sql/edr-index-sql/src/main/java/org/eclipse/edc/connector/store/sql/edr/schema/postgres/EndpointDataReferenceEntryMapping.java

```diff
@@ -24,6 +24,8 @@
  */
 public class EndpointDataReferenceEntryMapping extends TranslationMapping {
     public EndpointDataReferenceEntryMapping(EndpointDataReferenceEntryStatements statements) {
+        add("id", statements.getAssetIdColumn());
```

Comment:
I think we don't need the `id` in the database the "id" is the `transfer_process_id`

Reply:
if we want to make available the search with `id` i would map it to `transfer_process_id` column